prisma/factories/plus_voting_summary.ts

```diff
@@ -0,0 +1,21 @@
+import { Factory } from "fishery";
+import { PlusVotingSummary } from "@prisma/client";
+import prisma from "../client";
+import userFactory from './user';
+
+export default Factory.define<PlusVotingSummary>(({ params, onCreate }) => {
+  onCreate(plusVotingSummary => {
+    return prisma.plusVotingSummary.create({ data: plusVotingSummary });
+  });
+
+  return {
+    userId: userFactory.build().id,
```

Comment:
Just curious but what is happening with this line? 🤔

Reply:
This definitely needs further work 🙂 The idea here is that factories can automatically create other required associations, so you could theoretically do something like:
```typescript
// We can manually specify the User for a PlusVotingSummary if we want to...
const user = await userFactory.create();
const plusVotingSummary = await plusVotingSummaryFactory.create({user: user});
console.log(plusVotingSummary.userId); // user.id

// ...but by default, the PlusVotingSummary factory will _automatically_ create an associated User for us
const plusVotingSummary = await plusVotingSummaryFactory.create();
console.log(plusVotingSummary.userId); // a new User id that has automatically been saved to the DB
```

That said, I'm definitely not doing this correctly yet. I think I'll revert this bit of code and used a hard-coded value for now.